docs/index.md

```diff
@@ -133,9 +143,9 @@ implement this logic yourself.
 
 Instead, make updates to the desired data assets on a branch and then utilize a lakeFS merge to atomically expose the data to downstream consumers.
 
-To learn more about atomic cross-collection updates, check out [this video](https://www.youtube.com/watch?v=9OsjUvk5UJU) which describes the concept in more detail.
+To learn more about atomic cross-collection updates, check out [this video](https://www.youtube.com/watch?v=9OsjUvk5UJU) which describes the concept in more detail, along with [this notebook](https://github.com/treeverse/lakeFS-samples/blob/main/01-multi-table-transaction-consistency-deltalake-lakefs/lakeFS-DeltaLake-multi-table-transaction-consistency.ipynb).
 
-#### Troubleshooting production problems
+### Troubleshooting production problems
```

Comment:
I am missing the ML reproduction, whether here or as a separate bullet.

Reply:
added